interfaces/evse_board_support.yaml

```diff
@@ -0,0 +1,118 @@
+        type: number
+        minimum: 0
+        maximum: 100
+  pwm_off:
+    description: Turns PWM off (constant high voltage)
+  pwm_F:
+    description: Turns PWM off with Error F (constant negative voltage)
+  allow_power_on:
+    description: >-
+      Sets allow_power_on flag. If false, Relais must never be switched
+      on.
+    arguments:
+      value:
+        description: Flag and context
+        type: object
+        $ref: /evse_board_support#/PowerOnOff
+  force_unlock:
+    description: Force unlock motor lock (if e.g. enforced by OCPP. BSP should do regular lock/unlock by itself)
+    result:
+      description: Returns true if unlocking sequence was successfully executed
+      type: boolean
+  ac_switch_three_phases_while_charging:
+    description: >-
+      Special command to force switching between one and three phases while
+      charging is active. HW must go through some special sequence to ensure safe
+      operation.
+    arguments:
+      value:
+        description: 'True: switch to 3ph, False: switch to 1ph'
+        type: boolean
+  evse_replug:
+    description: >-
+      Special command initiate a virtual replug sequence without restarting
+      session. Emits a EvseReplugStarted event if supported and started. BSP will
+      take care to not emit other events such as CarPluggedIn/Out during that time.
+      Once finished it will emit a EvseReplugFinished. This is for testing purposes,
+      don't implement for production use.
+    arguments:
+      value:
+        description: Time in ms for the duration of the replug sequence
+        type: integer
+  ac_read_pp_ampacity:
+    description: >-
+      Read the current carrying capacity of the connected cable in ampere for AC charging
+      with a socket.
+      This has no meaning for DC or AC charging with a fixed attached cable, it does not 
+      need to be implemented and the returned value is not used in those cases.
+    result:
+      description: Returns the current carrying capacity of the connected cable
+      type: object
+      $ref: /board_support_common#/ProximityPilot
+  ac_set_overcurrent_limit_A:
+    description: >-
+      Sets the limit that should be used for over current detection in the BSP.
+      Do not use this to set PWM, use this value only for fast OC detection in the MCU.
+      This will set the actual charging value, so for 16A charging you will get 16A here,
+      you will need to add a margin in the BSP to avoid false triggers.
+    arguments:
+      value:
+        description: Ampere value
+        type: number
+vars:
+  event:
+    description: Event from ControlPilot signal/output relais
+    type: object
+    $ref: /board_support_common#/BspEvent
+  ac_nr_of_phases_available:
+    description: Instantaneous phase count available to car
+    type: integer
+    minimum: 1
+    maximum: 3
+  telemetry:
+    description: Other telemetry
+    type: object
+    $ref: /evse_board_support#/Telemetry
+  capabilities:
```

Comment:
Should this var be also used to notify the EvseManager that the BSP has captured a change in PP value?

Reply:
yes probably. At the moment there is just a getter (ac_read_pp_ampacity) which we need to read it at the right moment, but there should also be a var that publishes changes during the charging session (actually IEC only requires this to be read once on start, but we should also monitor during charging). Maybe it would be clearer to make it a separate var

Reply:
implemented